### ISSUE TITLE
Backticks don't disable formatting inside a message

### ISSUE THREAD

**Issue by eras** [wee-slack/wee-slack#566]
Not sure if it's possible to fix this with wee-slack or rather with weechat itself ;), but it seems backtick quoting doesn't appropriately disable ie. bold formatting inside the region. There is bug #354 which apparently can be fixed with wee-slack so I dared to file this as well :).

Ie. `\`hello \*bar\*\`` will result in `bar` being highlighted when the expected result is that the asterisks are visible and no formatting is performed.

**Comment by ToxicFrog** [comment]
Yeah, like #354 this one is my fault and fixing it will require a client side formatter that's smarter than the existing search-and-replace implementation.

Helpfully, the Slack API docs don't seem to include detailed semantics or a reference implementation for their markdown processor, although they do talk about ways to turn it off, so exactly replicating the behaviour of slack is probably a nonstarter, but it can definitely be less bad than it is.

**Comment by auscompgeek** [comment]
I've filed #567 as some quick fixes until we get a real fix.
